### DISCUSSION TITLE
Feature request: Moving points in map

### DISCUSSION THREAD

**Discussion by pablogglz** [Freika/dawarich#503] [Ideas]
Hi, as always, thank you for your work!
Here is the feature I'd like to request: in the map view, select a point and be able to move it to 'another' location; then, click save and the coordinates would change to that location.
I know this may be tricky or not intended, however many times the location that OwnTracks (the service I use) is not that accurate, and some points may be off by 3 to 20 meters when I'm inside a bar or a house, etc. In these cases, I'd like to be able to move the points to the actual location I was at. 

Maybe, and to prevent some 'unintended' use, the 'edit' time could be saved too.
Like, for edited points, showing:
- Timestamp dd/mm/yyyy, hh:mm 
- Latitude: xxx
- Longitude: xxx
- Original latitude and longitude: xxx xxx
- Edited: dd/mm/yyyy, hh:mm 

I would be very pleased if you could consider adding this feature.
Thanks again!

**Comment by dietrichmd** [comment]
I second this, along with the ability to edit/delete points from the map as well.

**Comment by Freika** [comment]
Points movement added in https://github.com/Freika/dawarich/releases/tag/0.22.4

**Reply by pablogglz** [reply]
Awesome!!

**Comment by dunc1ca** [comment]
When I use the drag and drop, and then refresh the map, it sometimes works and sometimes reverts back.
